pkg/controller/config/config_controller.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"github.com/tektoncd/operator/pkg/controller/rbac"
```

Comment:
Could you please move the flags into a separate package so that controllers do not have to depend on each other?

Reply:
done